lsp/angularls.lua

```diff
@@ -66,7 +66,7 @@ end
 ---@type vim.lsp.Config
 return {
   cmd = function(dispatchers, config)
-    local root_dir = (config and config.root) or fn.getcwd()
+    local root_dir = (config and config.root_dir) or fn.getcwd()
```

Comment:
Thanks 👍 btw, this is correctly flagged by LuaLS, we just haven't enabled it in CI yet (for this repo):

<img width="657" height="159" alt="Image" src="https://github.com/user-attachments/assets/e68a52b3-36c5-4c09-afdf-6e4e305de57d" />